x-pack/metricbeat/module/gcp/timeseries_metadata_collector.go

```diff
@@ -53,14 +57,19 @@ func (s *StackdriverTimeSeriesMetadataCollector) Metadata(ctx context.Context, i
 
 	ecs := mapstr.M{
 		ECSCloud: mapstr.M{
-			ECSCloudAccount: mapstr.M{
+			ECSCloudProject: mapstr.M{
 				ECSCloudAccountID:   accountID,
 				ECSCloudAccountName: accountID,
```

Comment:
I think we need to change the names of these constants; even if the values are correct ("id" and "name"), the name is misleading. 

They contain "CloudAccount" while we use them for the ECSCloudProject:

```go
	ECSCloudAccountID   = "id"
	ECSCloudAccountName = "name"
```

Reply:
The `ECSCloudAccountID` and `ECSCloudAccountName` is used in two places - for setting organization(id and name) and Project(id and name) hence the field name can be something generic like-  `ECSCloudID` and `ECSCloudName` ?